README.md

```diff
@@ -196,7 +196,7 @@ String or Array to specify address(es) for which sshd will bind. Corresponds to
 
 sshd_config_port
 ---------------------------
-String to specify listen port for sshd. Port option in sshd_config.
+String or Array to specify listen port[s] for sshd. Port option in sshd_config.
```

Comment:
I like to see the support for integers here :+1: 
Please add the good news here too :)